### ISSUE TITLE
Wrong/missing documentation for `attach export`

### ISSUE THREAD

**Issue by Atomk** [pdfcpu/pdfcpu#509]
There are some issues in the [documentation page](https://pdfcpu.io/attach/attach_extract) for `attach extract`.

The "arguments" table indicates that the "file" argument is required, but it is not.

This makes the "export all attachments" example potentially incorrect:
- docs: `pdfcpu attach extract ledger.pdf out *`
- what works: `pdfcpu attach extract ledger.pdf out`

On my system the first command above prints `attachment * not found`, I'm using latest release (0.3.13) so I have not tested on latest commit if the two command above both work in the same way.

I've also discovered that `attachment`, `attachments` and `attach` can be used interchangeably, maybe it could be noted in the [main attach page](https://pdfcpu.io/attach/attach).

(is there a way to contribute documentation? I haven't found any information about it)

**Comment by hhrutter** [comment]
Thanks for your valuable feedback!

My focus righ now is getting a release out .
As far as documentation we will be moving to a new site and I limit commits to the gh-pages branch to bugfixes.
All pdfcpu CLI commands may be abbreviated and will be executed as long as there is no ambiguity.

**Comment by Atomk** [comment]
I have tested with latest commit and I confirm the docs are wrong.
I corrected the error so other people won't have to waste time figuring out why the example doesn't work.

>All pdfcpu CLI commands may be abbreviated and will be executed as long as there is no ambiguity. 
The docs use multiple versions of a command in the same page, it was a bit confusing at first, I'm not sure if there is already something written about this but it looks like something worth mentioning.

Anyway thanks for the project!
